### ISSUE TITLE
Classification details page - Filter by code or name fails color contrast

### ISSUE THREAD

**Issue by tilen1976** [statisticsnorway/ssb-dataportal#584]
Ref Silktide:
```
Contrast ratio: 1.08:1 / 3:1
FAIL
To fix, do one of the following:

Make the control and its surroundings contrast strongly (at least 3:1).
Add a border to the control, which contrasts strongly with the control (at least 3:1).
```
